tools/clang/unittests/HLSLExec/LongVectorOps.def

```diff
@@ -125,6 +126,13 @@ OP_DEFAULT(UnaryMath, Log10, 1, "log10", "")
 OP_DEFAULT(UnaryMath, Log2, 1, "log2", "")
 OP_DEFAULT_DEFINES(UnaryMath, Frexp, 1, "TestFrexp", "", " -DFUNC_FREXP=1")
 
+OP(UnaryMath, IsFinite, 1, "TestIsFinite", "", " -DFUNC_TEST_ISFINITE=1",
```

Comment:
nit: These aren't really 'UnaryMath' operations. The name isn't currently used for anything outside of a human hint.
But I'm thinking 'FloatSpecial' would be better. My main nit is that these are logical operations, not math operations.